### PR TITLE
Verify if a volume is active.

### PR DESCRIPTION
**Pull request by danielebarbaro**
Hi @mattstauffer , this maybe solve #123 .

I used "maybe" because I've a doubt. 
At the moment I checked with `docker ps -a --filter volume=volume_name` that a volume exist and the result is:
Screenshot 2020-09-18 at 12 37 55

So when user chose an existing container takeout return an error until user chose a unique name container.

Do you want to handle ,in some way, also this behaviour?

Screenshot 2020-09-18 at 12 38 09

### CODE REVIEW

**Comment by mattstauffer** [comment]
I think the second screenshot's behavior is the target of #89, so we can ignore that for this PR. Reviewing the rest of the PR now. Thanks!

File: app/Shell/Docker.php [hunk 1]
```diff
@@ -61,6 +61,16 @@ public function allContainers(): Collection
         return $this->containerRawOutputToCollection($this->allContainersRawOutput());
     }
 
+    public function volumeIsAvailable(string $volumeName): bool
+    {
+        return $this->containerRawOutputToCollection($this->isVolumeUsedRawOutput($volumeName))->count() === 0;
+    }
+
+    public function usedVolumes(string $volumeName): Collection
```
Comment (mattstauffer) [inline app/Shell/Docker.php:69]: We can drop this, since it's not used, right?

File: app/Shell/Docker.php [hunk 2]
```diff
@@ -93,6 +103,12 @@ protected function allContainersRawOutput(): string
         return trim($this->shell->execQuietly($dockerProcessStatusString)->getOutput());
     }
 
+    protected function isVolumeUsedRawOutput(string $volumeName): string
```
Comment (mattstauffer) [inline app/Shell/Docker.php:106]: What if we rename this `listMatchingVolumesRawOutput`?

**Review by mattstauffer — CHANGES_REQUESTED** [review]
Great work! Just a few quick notes

**Comment by danielebarbaro** [comment]
Hi @mattstauffer , thanks for your review. A made some review and a merge main in my branch to avoid conflicts.

**Comment by danielebarbaro** [comment]
Hi @mattstauffer now PR is updated.
All tests pass.

This PR needs #157 .

**Comment by mattstauffer** [comment]
@danielebarbaro Great--taking a look now. Thanks!

**Comment by mattstauffer** [comment]
Looks great. Thanks so much @danielebarbaro!

**Pull request merged**